Dockerfile

```diff
@@ -90,12 +94,21 @@ WORKDIR /ocm-cli
 RUN curl -sSLf -O $OCM_URL
 
 # Checking the SHA-256 hash
-RUN echo $OCM_SHA256_CHECKSUM | sha256sum -c -
+RUN curl -sSLf $OCM_SHA256_CHECKSUM_URL | sha256sum -c -
 
 # Moving the validated ocm binary to /out and make it executable
 RUN mv ocm-linux-amd64 /out/ocm
 RUN chmod +x /out/ocm
 
+# Install golang with specific version
```

Comment:
I built a docker image based on your updated Dockerfile and run a container
```
docker run -it shawn-test /bin/bash
[root@cdb748e8bb57 ~]#
[root@cdb748e8bb57 ~]# go version
bash: go: command not found

echo $PATH
/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```
Looks  no `go path` included.  Can you double check? 

The `ocm version` and `yq --version` look all good.

Reply:
The go installed in the builder only, for building the hypershift, and it will copy the hypershift binary only to the final image

Reply:
oh, I see. Thanks. 
I can see the Acceptance Criteria also mentions: `An automated dependency upgrade system for binaries set up in the managed scripts repository.`  If it's an optional and not considering in this PR, then LGTM.

Reply:
As long as all the binaries/packages can be up to date all the time. I think it is kind of covered.